packages/server/test/support/fixtures/projects/e2e/cypress/integration/issue_3121_spec.coffee

```diff
@@ -0,0 +1,22 @@
+describe "issue #3121", ->
+  it "shows anchor href content inside aut-iframe when target='aut-iframe'", ->
+    cy
+      .visit("/iframe_with_src.html")
+      .get("iframe").then ($iframe) ->
+        return cy
+                .wrap($iframe.contents()
+                .find("body"))
+      .within ->
+        cy
+          .get("a")
+          .click();
+
+    cy
+      .get("body").then ($body) ->
+        expect($body).to.contain("hi")
+      .get("iframe")
+      .should("not.exist")
+        
+
+      
+       
```

Comment:
this could be moved into another spec file as an integration test, so we wouldn't have to run it as an e2e test which is slow

Reply:
Didn't know the exact location of the integration tests 👍 . 

Note: The amount of tests as well as their speed is awesomely impressive ;)